### PR TITLE
cgmnlm: new port in net

### PR DESCRIPTION
**Pull request by barracuda156**
#### Description

New port: https://git.sr.ht/~rwa/cgmnlm

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.6
Xcode 3.2

macOS 14.1.1
Xcode 15.0.1

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

**Comment by barracuda156** [comment]
Depends on https://github.com/macports/macports-ports/pull/21463

**Comment by reneeotten** [comment]
> Depends on https://github.com/macports/macports-ports/pull/21463

I just merged that one; please rebase

**Comment by barracuda156** [comment]
@reneeotten Done

File: net/cgmnlm/Portfile [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+name                cgmnlm
+version             1.6
+revision            0
+categories          net gemini
+license             GPL-3
+maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
+description         ${name} is a colorful Gemini line mode client
+long_description    {*}${description}
+
+homepage            https://git.sr.ht/~rwa/cgmnlm
```
Comment (herbygillot) [inline net/cgmnlm/Portfile:18]: I think we have a `sourcehut` portgroup.

Comment (barracuda156) [reply net/cgmnlm/Portfile:18]: Ok, I can try to use that in a while.

**Comment by barracuda156** [comment]
@herbygillot @reneeotten Sorry for a delay, travelling now. Move to PG, rebased.

**Pull request merged**